torch/nn/quantized/modules/conv.py

```diff
@@ -79,15 +94,10 @@ def __init__(self, weight, bias, scale, zero_point, dtype,
 
     @weak_script_method
     def forward(self, input):
-        return qF.conv2d(input=input,
-                         weight=self._packed_weight,
-                         bias=self.bias,
-                         stride=self.stride,
-                         padding=self.padding,
-                         dilation=self.dilation,
-                         groups=self.groups,
-                         padding_mode=self.padding_mode,
-                         scale=self.scale,
-                         zero_point=self.zero_point,
-                         dtype=self.dtype,
-                         prepacked=True)
+        if input.ndim != 4:
+            raise ValueError("Input shape must be `(N, C, H, W)`!")
+        return ops.quantized.fbgemm_conv2d(input,
```

Comment:
why do we avoid functional usage?

Reply:
That was a request, because we want the functional to always prepack the weights, while the module to prepack only in the constructor

Reply:
you mean functional will take a quantized weight rather than prepacked and quantized weight as input?

Reply:
Correct. We need to document it so that the user would be aware that the functional is less efficient :/